packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -35,6 +35,7 @@ class BottomSheetCell extends Component {
 		this.state = {
 			isEditingValue: props.autoFocus || false,
 			isScreenReaderEnabled: false,
+			localValue: props.value ? props.value : '',
```

Comment:
I think you could use here the [Nullish coalescing operator (`??`)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) to simplify the code.

Reply:
This line is no longer relevant, but this is a good callout about using a better logic operator nonetheless. 👍